Common/PM_datatypes.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+// please change this name
+enum WaypointType {PATH_FOLLOW = 0, ORBIT_FOLLOW, HOVER_WAYPOINT, TAKEOFF_WAYPOINT, LANDING_WAYPOINT, TRANSITION_WAYPOINT, TELEOP_MODE};
+
+struct CommandsFromSMToPM{
+    // please rename this
+    WaypointType waypoint_type;
+    WaypointsCommand telemetry_commands;
+    JetsonToZpMovementCommand jetson_commands; 
+    LandingInitiationCommand landing_initiation; 
+    LosSFData sf_data;
+};
+
+struct JetsonToZpMovementCommand {
+  float x;
+  float y;
+  float z;
+  float heading;
+};
+
+struct LandingInitiationCommand {
+    bool start_landing; 
+};
+
+struct JetsonMovementRequest {
+    bool request;
+}
+
+struct JetsonOdometryData {
+    LosSFData sensorData;
+}
+
+struct LosSFData_t {
```

Comment:
@Chrisc110 this is correct right?

Reply:
Maybe we can consider just grabbing the struct from LOS? what would be most convenient for developers?